donkeycar/templates/train.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env python3
 """
-Scripts to train a keras model using tensorflow.
```

Comment:
This has been refactored recently. This should be a lot easier now.

Reply:
Sorry - not sure I understand what changes to make here.

Reply:
I meant this got moved to `manage.py`. @DocGarbanzo can help you figure out how to move them over.

Reply:
Just pushed with a commit that moves this into base.py and reverts train.py